src/Microsoft.VisualStudio.Editors/PropPages/CodeAnalysisPropPage.Designer.vb

```diff
@@ -101,6 +101,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             Me.AnalysisLevelComboBox.FormattingEnabled = True
             resources.ApplyResources(Me.AnalysisLevelComboBox, "AnalysisLevelComboBox")
             Me.AnalysisLevelComboBox.Name = "AnalysisLevelComboBox"
+            Me.AnalysisLevelComboBox.AccessibleName = Me.AnalysisLevelLabel.Text
```

Comment:
This change will be reverted by the designer in future. We need to address this issue properly in `main`. cc @melytc

Reply:
@drewnoakes I'm curious about this comment - why would the designer revert this?